src/utils/__tests__/media-utils.spec.ts

```diff
@@ -0,0 +1,140 @@
+import mockAxios from "jest-mock-axios"
+
+import { config } from "@config/config"
+
+import {
+  MEDIA_DIRECTORY_NAME,
+  MEDIA_FILE_NAME,
+  MEDIA_FILE_SHA,
+  MEDIA_SITE_NAME,
+  IMAGE_FILE_PRIVATE_INPUT,
+  IMAGE_FILE_PUBLIC_INPUT,
+  PDF_FILE_PRIVATE_INPUT,
+  PDF_FILE_PUBLIC_INPUT,
+  SVG_FILE_PRIVATE_INPUT,
+  SVG_FILE_PUBLIC_INPUT,
+} from "@root/fixtures/media"
+
+import { getMediaFileInfo, isMediaFileOutput } from "../media-utils"
+
+const GITHUB_ORG_NAME = config.get("github.orgName")
+
+const mockGenericAxios = mockAxios.create()
+
+jest.mock("@utils/token-retrieval-utils", () => ({
+  getAccessToken: jest.fn().mockResolvedValue("token"),
+}))
+mockGenericAxios.get.mockResolvedValue({
+  data: "blahblah",
+  headers: {
+    "content-type": "blahblah",
+  },
+})
+
+describe("Media utils test", () => {
+  it("should return mediaUrl as raw github information for images in public repos", async () => {
+    const expectedResp = {
+      mediaUrl: `https://raw.githubusercontent.com/${GITHUB_ORG_NAME}/${MEDIA_SITE_NAME}/staging/${MEDIA_DIRECTORY_NAME}/${encodeURIComponent(
+        MEDIA_FILE_NAME
+      )}`,
+      name: MEDIA_FILE_NAME,
+      sha: MEDIA_FILE_SHA,
+      mediaPath: `${MEDIA_DIRECTORY_NAME}/${MEDIA_FILE_NAME}`,
+      type: "file",
```

Comment:
should this be `images`? also this seems to pass so...

Reply:
This is actually the github response! We're only expecting `file` or `dir`

Reply:
for clarification, github only returns either `file | dir` even for `images` right? i'm asking this because the test description said that this was for `... images in public repos`, so this confused me

Reply:
yup, github returns directory item information - any item which is a single item (even .png or .svg files) returns `file`, any subdirectory returns `dir`